application/cms/models.py

```diff
@@ -575,6 +579,19 @@ def review_token_expires_in(self, config):
         except Exception:
             return 0
 
+    @property
+    def has_known_statistical_errors(self) -> bool:
```

Comment:
Maybe add a comment here to describe that it returns true if any later minor version updates a statistical error, even if the version itself corrects a previous error?

Reply:
👍